docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -70,6 +73,21 @@ The data nodes that are part of a **scenario**, **pipeline** or **task** can be
     task.sales_history
     ```
 
+Data nodes can be retrieved by their config_id by calling `taipy.get_entities_by_config_id()^`, which will return
+a list of existing data nodes based on the config_id.
+
+!!! Example
+
+    ```python linenums="1"
+    import taipy as tp
+    import my_config
+
+    scenario = tp.create_scenario(my_config.monthly_scenario_cfg)
+
+    # Get all trained model data nodes by config_id
+    all_trained_model_dns = tp.get_entities_by_config_id(my_config.trained_model_cfg.id)
```

Comment:
```suggestion
    all_trained_model_dns = tp.get_entities_by_config_id("trained_model")
```

Should we represent the config_id here as a string?
It is easier to understand imo.